client/proxy.go

```diff
@@ -76,34 +68,22 @@ func (proxy *Proxy) Disable() error {
 // the proxy such as listen port and active toxics will be deleted as well. If you just wish to
 // stop and later enable a proxy, use `Enable()` and `Disable()`.
 func (proxy *Proxy) Delete() error {
-	httpClient := &http.Client{}
-	req, err := http.NewRequest("DELETE", proxy.client.endpoint+"/proxies/"+proxy.Name, nil)
-	if err != nil {
-		return err
+	err := proxy.client.delete("/proxies/" + proxy.Name)
+	if err == nil {
```

Comment:
for the sake of consistency wouldnt it be better to 

```
if err != nil {
  return fmt.Errorf("Delete: %w", err)
}
```

then on line 75, just `return nil`?

Reply:
make sense. Fixed.